### ISSUE TITLE
request(PERMISSIONS.IOS.FACE_ID) always return denied when press on allow button

### ISSUE THREAD

**Issue by amitkumarcoding** [zoontek/react-native-permissions#903]
### Before submitting a new issue

- [X] I tested using the latest version of the library, as the bug might be already fixed.
- [X] I tested using a [supported version](https://github.com/reactwg/react-native-releases/blob/main/docs/support.md) of react native.
- [X] I checked for possible duplicate issues, with possible answers.

### Bug summary

when i request the face ID permission request(PERMISSIONS.IOS.FACE_ID)
it always return denied when user press on allow button and when user press on don't allow it gave block

working fine on version "react-native-permissions": "^3.8.4",

### Library version

5.0.1

### Environment info

```shell
"react-native": "^0.71.10",
"react-native-permissions": "^5.0.1",
```

### Steps to reproduce

1. ask the permission
2. allow the permission
3. gaves denied always

### Reproducible sample code

```js
const FaceIdPermission = async () => {
  if (Platform.OS === 'ios') {
    // Request permission for Face ID
    const res = await request(PERMISSIONS.IOS.FACE_ID);
    console.log("🚀 ~ res ~ res:", res);
    return res === RESULTS.GRANTED;
  } else {
    return true; // Face ID is not applicable on Android
  }
};
```

**Comment by amitkumarcoding** [comment]
let res = await check(PERMISSIONS.IOS.FACE_ID);

this giving me granted always even before permission modal comes

**Comment by zoontek** [comment]
I identified the issue, will fix it and release asap.

**Comment by zoontek** [comment]
Fixed in [5.0.2](https://github.com/zoontek/react-native-permissions/releases/tag/5.0.2)

**Comment by amitkumarcoding** [comment]
Thanks @zoontek For quick fix. I have tested this and it is working as expected
